src/Components/ArticlesList.js

```diff
@@ -9,26 +9,65 @@ const ArticlesList = () => {
   const [isLoading, setIsLoading] = useState(false);
 
   const [searchParams, setSearchParams] = useSearchParams();
-  const topicQuery = searchParams.get('topic');
+  const topicQuery = searchParams.get("topic");
+  const sortByQuery = searchParams.get("sort_by");
+  const orderQuery = searchParams.get("order");
+
+  const setSortBy = (selectedSortBy) => {
+    const newSortParams = new URLSearchParams(searchParams);
+    newSortParams.set("sort_by", selectedSortBy);
+    setSearchParams(newSortParams);
+  };
+
+  const setOrder = (selectedOrder) => {
+    const newOrderParams = new URLSearchParams(searchParams);
+    newOrderParams.set("order", selectedOrder);
+    setSearchParams(newOrderParams);
+  };
 
   useEffect(() => {
     setIsLoading(true);
-    getArticles(topicQuery).then((articlesData) => {
+    getArticles(topicQuery, sortByQuery, orderQuery).then((articlesData) => {
       setArticles(articlesData);
       setIsLoading(false);
     });
-  }, [topicQuery]);
+  }, [topicQuery, sortByQuery, orderQuery]);
 
   return (
-    <main className="articlesList">
-      {isLoading ? (
-        <h2>Loading articles...</h2>
-      ) : (
-        articles.map((article) => {
-          return <ArticleCard key={article.article_id} article={article} />;
-        })
-      )}
-    </main>
+    <div className="articlesList">
```

Comment:
Nice work, as an optional piece of feedback I'd suggest ammending the select boxes to reflect the URL. ie if we copy and pasted an old link where the url had been modified